dask/dataframe/io/parquet/arrow.py

```diff
@@ -119,6 +153,59 @@ def _determine_dataset_parts(fs, paths, gather_statistics, filters, dataset_kwar
     return parts, dataset
 
 
+def _get_filesystem_and_path(passed_filesystem, path):
+    if passed_filesystem is None:
+        return resolve_filesystem_and_path(path, passed_filesystem)
```

Comment:
Again, would be reasonable to explicitly use fsspec here (although in practice, is `passed_filesystem` ever actually `None`?). pyarrow only supports a very limited set of file-system implementations.

Reply:
You're right - `fs` should already by known at this point. We can remove `_get_filesystem_and_path` altogether.